src/components/NFTPortfolio/State.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
```

Comment:
Can use `<component>` with `tag` prop with default is `div`
```suggestion
  <component :is="tag">
```